lib/mix/tasks/phx.gen.auth/injector.ex

```diff
@@ -162,24 +162,27 @@ defmodule Mix.Tasks.Phx.Gen.Auth.Injector do
   def app_layout_menu_code_to_inject(%Schema{} = schema, padding \\ 4, newline \\ "\n") do
     already_injected_str = "#{schema.route_prefix}/log_in"
 
+    base_tailwind_classes = "text-[0.8125rem] leading-6 text-zinc-900 hover:text-zinc-700"
+    link_tailwind_classes = "#{base_tailwind_classes} font-semibold hover:text-zinc-700"
+
```

Comment:
__Note:__ these updates cause the `test/mix/tasks/phx.gen.auth/injector_test.exs` tests to fail. The tests appear to be snapshot tests. Does the team we want to keep them that way? It feels bizarre to me anytime I have to copy the output from a new update and paste-update a test assertion, especially given the noise that all of these classes will end up adding to the snapshot.

My suggestion would be to update those tests so that they assert particular behaviors, which I'm happy to do, but if the team wants to keep the snapshot approach, happy to update those, too.

Reply:
@vanderhoop I was just taking a look at the tests and noticed that `hover:text-zinc-700` is defined in `base_tailwind_classes` and right here, so it ends up being duplicated in the `link_tailwind_classes` output:

```
text-[0.8125rem] leading-6 text-zinc-900 hover:text-zinc-700 font-semibold hover:text-zinc-700
                                         ^^^^^^^^^^^^^^^^^^^               ^^^^^^^^^^^^^^^^^^^
```

Reply:
🤦 . Great catch, @codebycaleb. I'll update the code and snapshots this weekend.

Reply:
@codebycaleb I've fixed the redundancy in https://github.com/vanderhoop/phoenix/commit/952421d8fc62d6ed8cf5dd060d3b2608029de4af. @jeregrine I think this PR is now ready to go